.github/workflows/main.yml

```diff
@@ -432,6 +439,18 @@ jobs:
       env:
         GH_TOKEN: ${{ github.token }}
 
+  fiber_tests:
+    name: wasmtime-fiber tests
+    runs-on: ubuntu-latest
+    env:
+      CARGO_NDK_VERSION: 2.12.2
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: true
+    - uses: ./.github/actions/install-rust
+    - run: cargo test -p wasmtime-fiber --no-default-features
```

Comment:
Oh when adding a new job like this you'll need to also add a "join" to the `ci-status` below so it's handled for failure-vs-not.

Reply:
Done!